File: app/src/main/java/com/chatserver/dd/chat/Cmd/View/IViewCmd.java
package com.chatserver.dd.chat.Cmd.View;

public interface IViewCmd {

    void setDataToRecyclerView(ViewStatusPlayers dataToRecyclerView);
    void onResponseFailure(Throwable throwable);
    void setEdittext(String playerName);
    String getEdittext();
    void showToast(String text);

    void showProgressBar();

    void hideProgressBar();

    void listIsEmpty();

    void listIsNotEmpty();
}

File: app/src/main/java/com/chatserver/dd/chat/Cmd/View/IViewCmd.kt
package com.chatserver.dd.chat.Cmd.View

import com.chatserver.dd.chat.Cmd.View.ViewStatusPlayers.P

interface IViewCmd {
    fun onResponseFailure(throwable: Throwable?)
    var edittext: String?
    fun showToast(text: String?)
    fun showProgressBar()
    fun hideProgressBar()
    fun listIsEmpty()
    fun listIsNotEmpty()
    fun setDataToRecyclerView(sorted: List<P?>)
}